jargons/xml.md

```diff
@@ -0,0 +1,9 @@
+---
+title: xml
+tags:
```

Comment:
burada sadece `genel` etiketini kullanmamiz yeterli olacaktir.

Reply:
@alpcanaydin tamamdır, sağ olasın geri dönüş için